src/app.ts

```diff
@@ -6,6 +7,9 @@ const app = new Koa();
 const router = createRouter();
 
 app.use(logger());
+app.use(cors({
+  origin: '*',
```

Comment:
Was putting the origin as `*` meant to be a safe-guard against allowing cross-origin authorisation requests? If the store needs to deal with JWTs or any `Authorization` header values the `Access-Control-Allow-Origin` header _must_ to match the `Origin` header on browsers. Setting the origin to this value is the default behaviour in koa cors (i.e. `app.use(cors())`). Just wanted to check if this value was on purpose (and if we should leave a comment?)

Reply:
Ah yeah, that's the point of `Vary: Origin`.

We will need authorization later down the line.